packages/govuk-frontend/src/govuk/components/details/details.mjs

```diff
@@ -11,151 +11,152 @@ const KEY_SPACE = 32
 
 /**
  * Details component
- *
- * @class
- * @param {Element} $module - HTML element to use for details
  */
-function Details ($module) {
-  if (!($module instanceof HTMLElement)) {
-    return this
-  }
-
-  /** @deprecated Will be made private in v5.0 */
-  this.$module = $module
+export class Details {
```

Comment:
Worth noting that this file is being deleted in #3766 – could remove the changes relating to this file to avoid potential unnecessary conflicts, unless that makes ESLint unhappy?

Reply:
I'm happy to fix conflicts 🤷‍♂️

Reply:
Let's try and merge the other PR first